test-cases/longevity/longevity-topology-changes-3h.yaml

```diff
@@ -18,6 +18,8 @@ n_loaders: 2
 n_monitor_nodes: 1
 
 instance_type_db: 'i4i.8xlarge'
+gce_instance_type_db: 'n1-highmem-32'
```

Comment:
same here.. but perhaps testing the 24h first, and if that one passes, we won't need to test this one

Reply:
For 24 we need to create custom instance type. There are 16 or 32 instances in gce (screenshots from gce website in the task). We were asked to test large instances that's why I put 32 cores here as well. 
If we want only a few pipelines with large instances with 32 cores please specify which pipelines to use with 32 and which with 16/8.

Reply:
Please , also check the memory and HDD

Reply:
Changed all 'n1's to 'n2's.
n2-highmem-32 has the same memory as i4i.8xlarge. 
We don't need a special run nor for this one neither the 24h.